sklearn/ensemble/_hist_gradient_boosting/common.pxd

```diff
@@ -19,8 +19,6 @@ cdef packed struct hist_struct:
 
 
 cdef packed struct node_struct:
```

Comment:
Maybe still add a comment

> struct for predictor tree nodes

to make  clear that this is related to predictor trees, not grower trees